submissions/BukkyOyetimehin/todo-list/src/components/InputTodo.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+
+class InputTodo extends Component {
+  state = {
+    title: "",
+  }
+  onChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
```

Comment:
Instead of `[e.target.name]: e.target.value` you can have `title: e.target.value` @Mayowa2020